pom.xml

```diff
@@ -30,5 +30,38 @@
             <artifactId>commons-io</artifactId>
             <version>2.6</version>
         </dependency>
+        <dependency>
+            <groupId>io.cucumber</groupId>
+            <artifactId>cucumber-testng</artifactId>
+            <version>3.0.2</version>
```

Comment:
please create property tag and move your versions to it

```
>     <properties>
>         <cucumber.version>3.0.2</cucumber.version>
           ...
>     </properties>
```

```
>         <dependency>
>             <groupId>io.cucumber</groupId>
>             <artifactId>cucumber-core</artifactId>
>             <version>${cucumber.version}</version>
>         </dependency>
>         <dependency>
>             <groupId>io.cucumber</groupId>
>             <artifactId>cucumber-java</artifactId>
>             <version>${cucumber.version}</version>
>         </dependency>
```

Reply:
done